terraform/ecs_alb/variables.tf

```diff
@@ -23,3 +23,7 @@ variable "health_check_path" {
 variable "vpc_id" {
   description = "ID of VPC to create ALB in"
 }
+
+variable "alarm_topic_arn" {
```

Comment:
Would it be worth having two different topics for alarms? I don't think we will treat 500 errors the same way as 400 errors